src/i18n/i18n.ts

```diff
@@ -142,8 +142,9 @@ const dict = {
         created: "Created",
         view: "View",
         id: "ID",
-        headline: "Privacy First, Non-Custodial Bitcoin Exchange",
-        subline: "Can't be evil. You are in control of your money. Always.",
+        headline: "Non-Custodial Bitcoin Bridge",
+        subline:
+            "Move your bitcoin between different layers while staying in full control. Powered by Atomic Swaps.",
```

Comment:
Why isn't Bitcoin capitalized here?

Reply:
Because it's bitcoin the asset vs Bitcoin the network/tech. We had this topic.